pyproject.toml

```diff
@@ -16,9 +16,10 @@ authors = [
   { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
 ]
 dependencies = [
-  "astropy>=5.0.6,!=5.1.0",
-  "gwcs>=0.18",
-  "numpy>=1.23.0"
+  "astropy>=5.3",
+  "gwcs>=0.20",
+  "numpy>=1.23.0",
+  "asdf>=2.14.4"
```

Comment:
Apologies for the delay @braingram . To ensure the `gwcs` tests pass, we need to use at least version 0.20 of `gwcs` because it includes the necessary [information](https://github.com/spacetelescope/gwcs/releases/tag/0.20.0#:~:text=Serialize%20and%20deserialize%20pixel_shape%20with%20asdf.%20%5B%23480%5D)( The addition of pixel_shape to the wcs schema used during the roundtrip serialization) introduced in that version. `gwcs` version 0.20 requires a minimum of `astropy` version 5.3, and the `asdf` version that passes the tests is at least 2.14.4.